### DISCUSSION TITLE
Copilot works on PyCharm but not in Visual Studio

### DISCUSSION THREAD

**Discussion by yanaiDia** [community/community#136196] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi,
I got the license to copilot, and used is successfully in PyCharm (JetBrains).
But in Visual Studio it doesn't work. I refreshed the credentials, and restarted the IDE, but it's not working.
Also the "open chat window" is disabled

VS 2022, 17.10.5

Any idea?
Thanks

**Comment by ayim** [comment]
Hello from VS team, sorry this hasn't been super smooth for ya. 
We'll need a little bit more information diagnose - mind filing a ticket via "Report a Problem" in Visual Studio?
![image](https://github.com/user-attachments/assets/e1a43bc4-3751-4047-8059-4ddf16087e17)

**Comment by yanaiDia** [comment, marked as answer]
Problem solved by removing the GitHub account from VS, and adding it back again
